src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -81,17 +81,17 @@ public void initBinder(WebDataBinder webdataBinder) {
     }
 
     @GetMapping(value = "/studies")
-    @Operation(summary = "Get all studies for a user")
+    @Operation(summary = "Get all studies")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The list of studies")})
-    public ResponseEntity<Flux<CreatedStudyBasicInfos>> getStudyList(@RequestHeader("userId") String userId) {
-        return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(studyService.getStudies(userId));
+    public ResponseEntity<Flux<CreatedStudyBasicInfos>> getStudyList() {
+        return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(studyService.getStudyList());
     }
 
     @GetMapping(value = "/study_creation_requests")
     @Operation(summary = "Get all study creation requests for a user")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The list of study creation requests")})
-    public ResponseEntity<Flux<BasicStudyInfos>> getStudyCreationRequestList(@RequestHeader("userId") String userId) {
-        Flux<BasicStudyInfos> studies = studyService.getStudyCreationRequests(userId);
+    public ResponseEntity<Flux<BasicStudyInfos>> getStudyCreationRequestList() {
```

Comment:
do we really want the list of all study creation request ?

Reply:
for maintenance purpose yes